### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1220] [Team Posts]
@spamatica 

Is the update of translation strings part of your release process? If not, then the docu is not right:

https://github.com/muse-sequencer/muse/wiki/Internationalisation#step-2

Because the users would have to update the strings themselves also for the releases, not only for the snapshots.

**Comment by spamatica** [comment]
Ah, yeah, it used to be but seems to have fallen into oblivion... probably since it was never added to the release steps in README.developer. ... and it was a while since we received a translation.

Maybe it is easier to update this document to always require step 2 ?

**Comment by kybos** [comment]
Yes, I guess so... I will change it.
